.github/workflows/sv_pipeline_docker.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Docker Images
+
+on: 
+  push:
+    branches:
+      - master
+    paths:
+      - 'src/**'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'src/**'
+
+jobs:
+  test_build:
+    runs-on: ubuntu-latest
+    name: Build GATK-SV Pipeline Docker Images
+    env:
+      PR_NUM: ${{ github.event.number }}
+      COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
+    strategy:
+      matrix:
+        python-version: ['3.8']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install termcolor
+
+      - name: Run build_docker.py
+        run: |
+          cd ./scripts/docker/
+          python build_docker.py --targets all --image-tag ${PR_NUM}-${COMMIT_SHA::8}
```

Comment:
Fine for now - `all` will not build MELT, which is the desired behavior in this case because the binaries need to be downloaded manually. It would be ideal to trigger a warning on the PR when a MELT dependency changes, if possible in a future PR.

Reply:
Agree we should also build `MELT`, I left it for future PRs as it requires hosted binaries (at least accessible to GitHub actions). I am not sure [if Github actions currently support warning, in addition to fail/pass](https://github.com/isaacs/github/issues/1465), so maybe we would leave checks/builds on `MELT` to continue to be manual for a while until we automate their builds?!

Reply:
Hm well doesn't look like there is much interest in warnings. But yes, let's leave MELT to manual builds for now.